lib/contentful/bootstrap/templates/json_template.rb

```diff
@@ -28,6 +30,12 @@ def entries
 
         private
 
+        def check_version
+          json_version = json.fetch('version', 0)
+          gem_major_version = Contentful::Bootstrap::VERSION.split('.').first.to_i
+          fail "JSON Templates Version Mismatch" unless gem_major_version == json_version
```

Comment:
Let's make this a bit more informative by telling people which version of bootstrap they need to use to import.